samcli/lib/utils/configuration.py

```diff
@@ -0,0 +1,23 @@
+""" Read info from config.json file"""
```

Comment:
We should call this something more specific than configuration. This can be confused with GlobalConfig and toml config. Maybe build-configuration, release-configuration, or something else.

Reply:
rename it to runtime_config.json